cpp/ql/src/semmle/code/cpp/Type.qll

```diff
@@ -1001,7 +1003,7 @@ class ArrayType extends DerivedType {
  * A GNU/Clang vector type.
  *
  * In both Clang and GNU compilers, vector types can be introduced using the
- * __attribute__((vector_size(byte_size))) syntax. The Clang compiler also
+ * `__attribute__((vector_size(byte_size)))` syntax. The Clang compiler also
  * allows vector types to be introduced using the ext_vector_type,
  * neon_vector_type, and neon_polyvector_type attributes (all of which take
  * an element type rather than a byte size).
```

Comment:
Also back-tick `ext_vector_type`, `neon_vector_type` and `neon_polyvector_type`.